library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -300,7 +310,16 @@ class Client() {
         var dbPath = ""
         val v3Client: FfiXmtpClient? =
             if (isAlphaMlsEnabled(options)) {
-                val alias = "xmtp-${options!!.api.env}-${accountAddress.lowercase()}"
+                var inboxId = getInboxIdForAddress(
```

Comment:
Should `createClient` take this new `inboxId`? It appears the signature for createClient actually didn't change?

Reply:
`createClient` still takes the address of the wallet. The only big change is the dbPath, which should be something like `$inbox_id-$env`

Reply:
Should we still allow the integrator to pass a dbPath?

Reply:
What if we let them pass in a `dbFolder`?